package.json

```diff
@@ -10,9 +10,11 @@
 		"prepare": "husky"
 	},
 	"dependencies": {
+		"@hono/zod-validator": "^0.2.2",
```

Comment:
```suggestion
```
it looks like `@hono/zod-validator` was added but never used

Reply:
oops